package.json

```diff
@@ -18,6 +18,10 @@
     "lint": "eslint ./src --ext .ts",
     "lint:fix": "eslint ./src --ext .ts --fix",
     "test": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:e2e": "jest --config ./test/jest-e2e.json",
```

Comment:
we can move the `jest.config.js` file into `test/` folder. alternatively, we can leave it where it is an simply use that instead (in which case, we can delete the `jest-e2e.json` files)

Reply:
Top level jest file is for Unit tests. We don't have them at the moment, but they should be close to class files and not in /test folder. 

In new nestJs samples jest.config.js is named as jest.json and in json format.

Reply:
ok cool. can we use the same naming convention as above though (i.e. `jest.api-test.js/json`? we can use either `*.js` or `*.json` - we just need to specify where the config file is in package.json

Reply:
Renamed in this repo. We can also rename e2e-spec to api-test in nestjs-template